int16 encoder_L
int16 encoder_R
